vector/src/main/java/im/vector/app/features/home/room/list/home/invites/InvitesViewModel.kt

```diff
@@ -129,8 +142,26 @@ class InvitesViewModel @AssistedInject constructor(
                 sortOrder = RoomSortOrder.ACTIVITY
         )
 
-        setState {
-            copy(pagedList = pagedList)
-        }
+        pagedList.asFlow()
+                .map {
+                    if (it.isEmpty()) {
+                        InvitesContentState.Empty(
+                                title = stringProvider.getString(R.string.invites_empty_title),
+                                image = drawableProvider.getDrawable(R.drawable.ic_invites_empty),
+                                message = stringProvider.getString(R.string.invites_empty_message)
+                        )
+                    } else {
+                        invitesCount = it.loadedCount
```

Comment:
This will work since we will test only if this is equal to 1, but I think it's more correct to use `it.size()`

Reply:
`it.size()` will also return placeholders (if there are any), while here we are interested in exact count without them

Reply:
from my understanding, the exact count is returned by `size()`. `loadedCount` does not include placeholders (= not loaded data, but which exist). If we have for instance 120 invites, there is maybe a risk that `loadedCount` return 0, or 1 (and so we have a bug), while `size()` returns 120, during the time the data are not yet loaded.
Anyway the risk is very low I think, and probably temporary, since more data will be loaded to be displayed.